cluster/gce/container-linux/configure-helper.sh

```diff
@@ -516,7 +516,9 @@ function start-kubelet {
     if [[ "${REGISTER_MASTER_KUBELET:-false}" == "true" ]]; then
       flags+=" --api-servers=https://${KUBELET_APISERVER}"
       flags+=" --register-schedulable=false"
-      flags+=" --register-with-taints=node.alpha.kubernetes.io/ismaster=:NoSchedule"
+      if [[ "${ENABLE_MASTER_NOSCHEDULE_TAINT:-false}" == "true" ]]; then
+        flags+=" --register-with-taints=node.alpha.kubernetes.io/ismaster=:NoSchedule"
```

Comment:
Kubelet uses .node.spec.taints. The alpha indicates that the name of the key is alpha, like we do with label and annotation keys.